Classes/Core/Testbase.php

```diff
@@ -529,7 +623,12 @@ public function setUpPackageStates(
      */
     private function isPackageArtifactNeeded(): bool
     {
-        return defined('TYPO3_COMPOSER_MODE') && class_exists(PackageCacheEntry::class);
+        return false;
+
+        // For the time being we do override composer mode and this code is not
+        // needed anymore but kept here for later usage once real composer tests
+        // get included.
+        //return defined('TYPO3_COMPOSER_MODE') && class_exists(PackageCacheEntry::class);
```

Comment:
We have version control. If we don't need code any more, we can safely remove it 😅